docs/pages/roadmap/whats_new.js

```diff
@@ -119,24 +134,30 @@ export default function Blog(): Node {
         ))}
       </RadioGroup>
 
-      {blogPosts.digests.map((digest) => (
-        <MainSection key={`digest-${digest.week}`} name={`Week of ${digest.week}`}>
-          <Flex
-            direction="column"
-            gap={{
-              row: 0,
-              column: 12,
-            }}
-          >
-            {digest.posts.map(
-              (post) =>
-                (filter === 'All' || post.audience.includes(filter)) && (
-                  <PostLayout key={`post-${post.title}`} {...post} />
-                ),
-            )}
+      <Flex direction="column" gap={12}>
+        {filteredDigests.map(({ month, week, posts }, i) => (
+          <Flex key={`digest-${week ?? month}`} direction="column" gap={12}>
+            {i > 0 && <Divider />}
```

Comment:
adding a divider and additional spacing between weeks/months to make it easier to scan